### ISSUE TITLE
Setup Tenon.io for accessiblity testing

### ISSUE THREAD

**Issue by nschonni** [wet-boew/wet-boew#7004]
It looks like they have an open source plan now http://us2.campaign-archive1.com/?u=db90adfcd71356341118b0a41&id=de77e18b90&e=4de96f8850
Probably should be setup under the @wet-boew-bot.
This could close #471

/cc @LaurentGoderre

**Comment by hsrudnicki** [comment]
Please update the status of this issue whether this issue can be closed or otherwise?

**Comment by hsrudnicki** [comment]
Please update this issue's status when possible, thanks

**Comment by hsrudnicki** [comment]
Make "Going Stale" or otherwise?
Please keep this issue's status updated with the other referenced (471), thank you

**Comment by karlgroves** [comment]
Just to confirm: you can use Tenon for free for your open source project. ;-)

**Comment by hsrudnicki** [comment]
Can be this issue closed as resolved or otherwise? Please update this issue's status, thx

**Comment by nschonni** [comment]
Tenon has shut down and been merged with other products I believe
